emission/core/wrapper/trip.py

```diff
@@ -88,7 +114,15 @@ def get_distance(self):
         return cm.calDistance(self.trip_start_location, self.trip_end_location, True)
 
     def save_to_db(self):
-        pass
+        db = edb.get_trip_db()
+        for section in self.sections:
+            section.save_to_db()
+        db.insert({"user_id": self.user_id, "trip_id": self.trip_id, "sections": range(len(self.sections)), "trip_start_datetime": self.start_time,
```

Comment:
wait! why are we inserting trips into the database from here? The only place where we should be inserting trips into the database is while processing the data from the phone. I don't see any restructuring of the code that processes data from the phone to use this.... Is this to store representatives? If so, you should use the canonical trips.